Gemfile

```diff
@@ -1,9 +1,11 @@
+# TOOD: figure out how to remove redundancy between this and rails4.2.gemfile
+
```

Comment:
this whole file can just become `eval_gemfile 'gemfiles/rails4.2.gemfile'`

Reply:
I tried that! it blows up because the relative path used in the inner gemspec.